bin/compile

```diff
@@ -35,16 +38,38 @@ cp -n $BP_DIR/scripts/Procfile $BUILD_DIR/Procfile
 echo "-----> Copying static.json to application slug"
 cp -n $BP_DIR/scripts/static.json $BUILD_DIR/static.json
 
-echo "-----> Installing Bower package manager"
 cd $BUILD_DIR
-npm install bower
+if [ -d "./bower_components" ]; then
```

Comment:
If we want to support bower component installation here, then we should probably cache it too.